plugins/autodoc/autodoc-converters/src/test/java/org/eclipse/edc/plugins/autodoc/markdown/MarkdownManifestRendererTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
```

Comment:
Probably the copyright header is outdated :)

Reply:
yep - copyright profile in IntelliJ was out of date